tailwind.config.js

```diff
@@ -24,6 +25,8 @@ module.exports = {
     },
     backgroundImage: {
       'hero-shapes': "url('/img/header-shapes-mobile.png')",
+      'mobile-footer-shapes':
+        "url('/img/footer/contact form shapes-mobile@2x.png')",
```

Comment:
[OPTIONAL] 🤓 Good job adding the required background images. I see you are using TailwindCSS, nice effort. However, I suggest you use the relative path `./path-to-media` when linking your assets. That way you keep it consistent between dev environments. Take a look at this [live version](https://raw.githack.com/martinkarugaba/my-portfolio/add-form/index.html) of your work. Some environments have problems handling non-relative paths.